gr-supacars.lwr

```diff
@@ -20,7 +20,11 @@
 inherit: empty
 category: application
 depends:
+- gnuradio
 - rtl-sdr
+- gr-pyqt
+- wget
+- pip
```

Comment:
Why are pip and wget dependencies of this recipe?

Reply:
pip and wget are used to install some additional packages